mantle/cmd/kola/testiso.go

```diff
@@ -618,7 +622,9 @@ func runTestIso(cmd *cobra.Command, args []string) (err error) {
 			case "ibft":
 				butane_config = strings.ReplaceAll(iscsi_butane_config, "COREOS_INSTALLER_KARGS", "--append-karg rd.iscsi.firmware=1")
 			case "manual":
-				butane_config = strings.ReplaceAll(iscsi_butane_config, "COREOS_INSTALLER_KARGS", "--append-karg netroot=iscsi:10.0.2.15::::iqn.2024-05.com.coreos")
+				butane_config = strings.ReplaceAll(iscsi_butane_config, "COREOS_INSTALLER_KARGS", "--append-karg netroot=iscsi:10.0.2.15::::iqn.2024-05.com.coreos:0")
+			case "ibft-with-multipath":
+				butane_config = strings.ReplaceAll(iscsi_butane_config, "COREOS_INSTALLER_KARGS", "--append-karg rd.iscsi.firmware=1 --append-karg rd.multipath=default --append-karg root=/dev/disk/by-label/dm-mpath-root")
```

Comment:
Interesting. This normally should also require `--append-karg rw` as per https://github.com/openshift/os/blob/master/docs/faq.md#q-does-rhcos-support-multipath-on-the-primary-disk. Wonder why it's not required here.

Reply:
In my original testing I always added `rw` but then when I got here I was challenging every kernel argument and testing without them and it still passed the test. I went back and removed it in my original setup and it wasn't needed there either from what I could tell.

Reply:
followup in https://github.com/coreos/coreos-assembler/pull/3797